.travis.yml

```diff
@@ -24,26 +18,47 @@ branches:
 matrix:
     fast_finish: true
     include:
+          # Minimum supported PHP and Symfony version
```

Comment:
Here PHP is the highest available version

Reply:
Yes, see https://github.com/symfony/symfony-docs/pull/8701#discussion_r152016042

Reply:
So maybe a comment like *Test latest PHP version with lowest dependencies* would be more accurate.